code/controllers/subsystem/job.dm

```diff
@@ -560,6 +571,19 @@ SUBSYSTEM_DEF(job)
 	else if((job.auto_deadmin_role_flags & DEADMIN_POSITION_SILICON) && ((CONFIG_GET(flag/auto_deadmin_silicons) && !timegate_expired) || (C.prefs?.toggles & DEADMIN_POSITION_SILICON))) //in the event there's ever psuedo-silicon roles added, ie synths.
 		return C.holder.auto_deadmin()
 
+/datum/controller/subsystem/job/proc/setup_assistant_positions()
+	var/datum/job/job = SSjob.GetJob(new_player_role)
+	if(!job)
+		CRASH("setup_assistant_positions(): Assistant job is missing")
+
+	var/assistant_ratio = CONFIG_GET(number/assistant_scaling_coeff)
+	if (assistant_ratio <= 0 || job.spawn_positions <= 0)
+		return
+	var/assistant_positions = max(3, max(job.spawn_positions, round(unassigned.len / assistant_ratio))) //with a ratio of 8, we'd need a pop of more than 24 to have more than 3 assistants (we'll always have at least 3 of them)
```

Comment:
this doesnt seem to get upped when the server reaches higher pop counts and looks like it will count nukies/wizards which you theoretically could metagame

Reply:
i just used the same logic used for sec officers. So if nobody metagamed with sec officers numbers, you won't metagame with assistant numbers (wich are more random due to new players always able to select them)

Reply:
snowflake antag assignment should happen before job assignment, iirc. so unassigned.len should exclude nukies/wiznerds. unless im talking out of my ass.